definitions/variable/macro-economy/tag_economic_sectors_level-1.yaml

```
@@ -0,0 +1,11 @@
+# Economic sectors based on the International Standard Industrial Classification
+# of All Economic Activities (ISIC) Rev.4
+# https://unstats.un.org/unsd/demographic-social/census/documents/isic_rev4.pdf
+- Economic Sectors Level 1:
+    - Agriculture:
+        description: the aggregated agricultural sectors (ISIC Rev.4; Section A)
```

This is not checked by nomenclature because it's not used, but should be made consistent.
```suggestion
        description: the aggregated agricultural sectors (ISIC Rev.4 - Section A)
```